src/app/pages/forms-and-validation/phone-number-format/phone-number-format.component.scss

```diff
@@ -0,0 +1,42 @@
+@import '~@pxblue/colors/palette.scss';
+
+section.mat-typography {
+    padding-top: 40px;
+    max-width: 480px;
+    margin: 0 auto;
+    .phone-number-form {
+        display: flex;
+        .country-code-select {
+            max-width: 112px;
+        }
+        .phone-number-input {
+            margin-left: 16px;
+            flex: 1;
+        }
+    }
+}
+
+::ng-deep .pxb-drawer-layout-nav-content {
+    background-color: map-get($pxb-white, 50);
+}
+
+::ng-deep .mat-form-field.mat-focused.mat-primary .mat-select-arrow {
+    border-top: 0;
+    border-bottom: 5px solid;
+}
+
+::ng-deep .cdk-overlay-container .cdk-overlay-pane .selectPanel {
+    min-width: 112px !important;
```

Comment:
Update this to match the updated select input width
```suggestion
    min-width: 135px !important;
```
It would be ideal to increase specificity here to avoid the `!important` flag as well.

Reply:
It didn't work without important flag, since mat-select-panel calculates width and applies on the fly. 
![image](https://user-images.githubusercontent.com/10433274/130204078-ea2698f7-3336-4b6c-b85f-823aa28a5aa8.png)